CONTRIBUTING.md

```diff
@@ -30,6 +30,8 @@ It's helpful to run the Coq test suite with `make test-suite` before submitting
 
 Don't be alarmed if the pull request process takes some time. It can take a few days to get feedback, approval on the final changes, and then a merge. Coq doesn't release new versions very frequently so it can take a few months for your change to land in a released version. That said, you can start using the latest Coq `master` branch to take advantage of all the new features, improvements, and fixes.
 
+Whitespace discipline (do not indent using tabs, no trailing spaces) is checked by Travis (using `git diff --check`). You can ensure that your commits do not introduce errors by adding the [`dev/tools/pre-commit`](/dev/tools/pre-commit) script to your `.git/hooks/` directory.
+
```

Comment:
Maybe add "just running configure" will add this, if you do not have a pre-existing pre-commit hook.